src/transaction/TransactionController.ts

```diff
@@ -636,7 +636,10 @@ export class TransactionController extends BaseController<
 
       const txNonce =
         nonce ||
-        (await query(this.ethQuery, 'getTransactionCount', [from, 'pending']));
```

Comment:
These changed because there is now a new type parameter on `query` that governs the type of data that it returns (as unfortunately due to the generic nature of `query`, we can't figure it out). The behavior is the same though, these are just type annotations.